#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile:
#1 transferring dockerfile: 3.12kB done
#1 DONE 0.4s

#2 [internal] load metadata for docker.io/library/python:3.10
#2 DONE 0.0s

#3 [internal] load .dockerignore
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [1/5] FROM docker.io/library/python:3.10
#4 DONE 0.0s

#5 [2/5] RUN mkdir -p ~/.pip && touch ~/.pip/pip.conf &&     echo "[global]" >> ~/.pip/pip.conf && echo "index-url=http://simple/" >> ~/.pip/pip.conf && echo "[install]" >> ~/.pip/pip.conf && echo "trusted-host=pypi..org" >> ~/.pip/pip.conf &&     apt-get update && apt-get install -y curl &&     curl -sSL https://install.python-poetry.org | python -
#5 CACHED

#6 [3/5] RUN pip install pytest pytest-xdist &&     pip install pipdeptree &&     git clone https://github.com/taobojlen/django-zeal.git &&     mkdir /repo &&     git config --global --add safe.directory /repo &&     cp -r /django-zeal/. /repo && rm -rf /django-zeal/ &&     rm -rf /django-zeal &&     cd /repo && git checkout 232987c7ad3f33a1ad294fa2bd2dcb973e949cd2 &&     pip install asgiref==3.8.1 &&     pip install backports-tarfile==1.2.0 &&     pip install build==1.2.1 &&     pip install certifi==2024.6.2 &&     pip install cffi==1.17.1 &&     pip install charset-normalizer==3.3.2 &&     pip install django==4.2.13 &&     pip install django-stubs==5.0.2 &&     pip install django-stubs-ext==5.0.2 &&     pip install docutils==0.21.2 &&     pip install exceptiongroup==1.2.2 &&     pip install factory-boy==3.3.0 &&     pip install faker==26.0.0 &&     pip install filelock==3.16.1 &&     pip install idna==3.7 &&     pip install importlib-metadata==8.5.0 &&     pip install iniconfig==2.0.0 &&     pip install jaraco-classes==3.4.0 &&     pip install jaraco-context==5.3.0 &&     pip install jaraco-functools==4.0.1 &&     pip install keyring==25.2.1 &&     pip install markdown-it-py==3.0.0 &&     pip install mdurl==0.1.2 &&     pip install more-itertools==10.3.0 &&     pip install nh3==0.2.17 &&     pip install nodeenv==1.9.1 &&     pip install packaging==24.1 &&     pip install pkginfo==1.10.0 &&     pip install pluggy==1.5.0 &&     pip install pycparser==2.22 &&     pip install pygments==2.18.0 &&     pip install pyproject-hooks==1.1.0 &&     pip install pyright==1.1.369 &&     pip install pytest==8.2.2 &&     pip install pytest-codspeed==3.0.0 &&     pip install pytest-django==4.8.0 &&     pip install pytest-mock==3.14.0 &&     pip install pytest-random-order==1.1.1 &&     pip install python-dateutil==2.9.0.post0 &&     pip install readme-renderer==43.0 &&     pip install requests==2.32.3 &&     pip install requests-toolbelt==1.0.0 &&     pip install rfc3986==2.0.0 &&     pip install rich==13.9.4 &&     pip install ruff==0.5.0 &&     pip install six==1.16.0 &&     pip install sqlparse==0.5.0 &&     pip install tomli==2.1.0 &&     pip install twine==5.1.1 &&     pip install types-pyyaml==6.0.12.20240311 &&     pip install typing-extensions==4.12.2 &&     pip install urllib3==2.2.2 &&     pip install zipp==3.21.0
#6 1.747 Looking in indexes: http://simple/
#6 1.906 Collecting pytest
#6 1.929   Downloading http://packages/pytest/pytest-8.3.4-py3-none-any.whl (343 kB)
#6 1.954      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 343.1/343.1 kB 17.4 MB/s eta 0:00:00
#6 2.027 Collecting pytest-xdist
#6 2.046   Downloading http://packages/pytest-xdist/pytest_xdist-3.6.1-py3-none-any.whl (46 kB)
#6 2.057      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.1/46.1 kB 4.8 MB/s eta 0:00:00
#6 2.126 Collecting pluggy<2,>=1.5
#6 2.149   Downloading http://packages/pluggy/pluggy-1.5.0-py3-none-any.whl (20 kB)
#6 2.183 Collecting iniconfig
#6 2.202   Downloading http://packages/iniconfig/iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
#6 2.262 Collecting tomli>=1
#6 2.280   Downloading http://packages/tomli/tomli-2.2.1-py3-none-any.whl (14 kB)
#6 2.353 Collecting packaging
#6 2.372   Downloading http://packages/packaging/packaging-24.2-py3-none-any.whl (65 kB)
#6 2.384      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.5/65.5 kB 7.4 MB/s eta 0:00:00
#6 2.432 Collecting exceptiongroup>=1.0.0rc8
#6 2.460   Downloading http://packages/exceptiongroup/exceptiongroup-1.2.2-py3-none-any.whl (16 kB)
#6 2.517 Collecting execnet>=2.1
#6 2.537   Downloading http://packages/execnet/execnet-2.1.1-py3-none-any.whl (40 kB)
#6 2.545      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.6/40.6 kB 8.3 MB/s eta 0:00:00
#6 2.687 Installing collected packages: tomli, pluggy, packaging, iniconfig, execnet, exceptiongroup, pytest, pytest-xdist
#6 3.291 Successfully installed exceptiongroup-1.2.2 execnet-2.1.1 iniconfig-2.0.0 packaging-24.2 pluggy-1.5.0 pytest-8.3.4 pytest-xdist-3.6.1 tomli-2.2.1
#6 3.292 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#6 3.431 
#6 3.431 [notice] A new release of pip is available: 23.0.1 -> 24.3.1
#6 3.431 [notice] To update, run: pip install --upgrade pip
#6 4.277 Looking in indexes: http://simple/
#6 4.359 Collecting pipdeptree
#6 4.382   Downloading http://packages/pipdeptree/pipdeptree-2.24.0-py3-none-any.whl (32 kB)
#6 4.522 Collecting pip>=24.2
#6 4.533   Using cached http://packages/pip/pip-24.3.1-py3-none-any.whl (1.8 MB)
#6 4.551 Requirement already satisfied: packaging>=24.1 in /usr/local/lib/python3.10/site-packages (from pipdeptree) (24.2)
#6 4.655 Installing collected packages: pip, pipdeptree
#6 4.655   Attempting uninstall: pip
#6 4.656     Found existing installation: pip 23.0.1
#6 4.820     Uninstalling pip-23.0.1:
#6 4.971       Successfully uninstalled pip-23.0.1
#6 6.316 Successfully installed pip-24.3.1 pipdeptree-2.24.0
#6 6.317 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#6 6.532 Cloning into 'django-zeal'...
#6 8.385 Note: switching to '232987c7ad3f33a1ad294fa2bd2dcb973e949cd2'.
#6 8.385 
#6 8.385 You are in 'detached HEAD' state. You can look around, make experimental
#6 8.385 changes and commit them, and you can discard any commits you make in this
#6 8.385 state without impacting any branches by switching back to a branch.
#6 8.385 
#6 8.385 If you want to create a new branch to retain commits you create, you may
#6 8.385 do so (now or later) by using -c with the switch command. Example:
#6 8.385 
#6 8.385   git switch -c <new-branch-name>
#6 8.385 
#6 8.385 Or undo this operation with:
#6 8.385 
#6 8.385   git switch -
#6 8.385 
#6 8.385 Turn off this advice by setting config variable advice.detachedHead to false
#6 8.385 
#6 8.385 HEAD is now at 232987c chore: update README (#45)
#6 9.051 Looking in indexes: http://simple/
#6 9.104 Collecting asgiref==3.8.1
#6 9.128   Downloading http://packages/asgiref/asgiref-3.8.1-py3-none-any.whl (23 kB)
#6 9.182 Collecting typing-extensions>=4 (from asgiref==3.8.1)
#6 9.206   Downloading http://packages/typing-extensions/typing_extensions-4.12.2-py3-none-any.whl (37 kB)
#6 9.249 Installing collected packages: typing-extensions, asgiref
#6 9.349 Successfully installed asgiref-3.8.1 typing-extensions-4.12.2
#6 9.350 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 10.08 Looking in indexes: http://simple/
#6 10.11 Collecting backports-tarfile==1.2.0
#6 10.11   Using cached http://packages/backports-tarfile/backports.tarfile-1.2.0-py3-none-any.whl (30 kB)
#6 10.15 Installing collected packages: backports-tarfile
#6 10.19 Successfully installed backports-tarfile-1.2.0
#6 10.19 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 10.85 Looking in indexes: http://simple/
#6 10.90 Collecting build==1.2.1
#6 10.92   Downloading http://packages/build/build-1.2.1-py3-none-any.whl (21 kB)
#6 10.94 Requirement already satisfied: packaging>=19.1 in /usr/local/lib/python3.10/site-packages (from build==1.2.1) (24.2)
#6 10.97 Collecting pyproject_hooks (from build==1.2.1)
#6 10.97   Using cached http://packages/pyproject-hooks/pyproject_hooks-1.2.0-py3-none-any.whl (10 kB)
#6 10.97 Requirement already satisfied: tomli>=1.1.0 in /usr/local/lib/python3.10/site-packages (from build==1.2.1) (2.2.1)
#6 11.01 Installing collected packages: pyproject_hooks, build
#6 11.06 Successfully installed build-1.2.1 pyproject_hooks-1.2.0
#6 11.06 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 11.76 Looking in indexes: http://simple/
#6 11.82 Collecting certifi==2024.6.2
#6 11.84   Downloading http://packages/certifi/certifi-2024.6.2-py3-none-any.whl (164 kB)
#6 11.90 Installing collected packages: certifi
#6 11.92 Successfully installed certifi-2024.6.2
#6 11.92 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 12.63 Looking in indexes: http://simple/
#6 12.90 Collecting cffi==1.17.1
#6 12.91   Using cached http://packages/cffi/cffi-1.17.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (446 kB)
#6 12.93 Collecting pycparser (from cffi==1.17.1)
#6 12.93   Using cached http://packages/pycparser/pycparser-2.22-py3-none-any.whl (117 kB)
#6 12.98 Installing collected packages: pycparser, cffi
#6 13.25 Successfully installed cffi-1.17.1 pycparser-2.22
#6 13.25 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 14.01 Looking in indexes: http://simple/
#6 14.21 Collecting charset-normalizer==3.3.2
#6 14.24   Downloading http://packages/charset-normalizer/charset_normalizer-3.3.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (142 kB)
#6 14.30 Installing collected packages: charset-normalizer
#6 14.35 Successfully installed charset-normalizer-3.3.2
#6 14.35 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 15.06 Looking in indexes: http://simple/
#6 15.27 Collecting django==4.2.13
#6 15.53   Downloading http://packages/django/Django-4.2.13-py3-none-any.whl (8.0 MB)
#6 15.60      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.0/8.0 MB 141.9 MB/s eta 0:00:00
#6 15.67 Requirement already satisfied: asgiref<4,>=3.6.0 in /usr/local/lib/python3.10/site-packages (from django==4.2.13) (3.8.1)
#6 15.70 Collecting sqlparse>=0.3.1 (from django==4.2.13)
#6 15.72   Downloading http://packages/sqlparse/sqlparse-0.5.3-py3-none-any.whl (44 kB)
#6 15.74 Requirement already satisfied: typing-extensions>=4 in /usr/local/lib/python3.10/site-packages (from asgiref<4,>=3.6.0->django==4.2.13) (4.12.2)
#6 15.82 Installing collected packages: sqlparse, django
#6 18.11 Successfully installed django-4.2.13 sqlparse-0.5.3
#6 18.11 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 18.86 Looking in indexes: http://simple/
#6 18.93 Collecting django-stubs==5.0.2
#6 19.06   Downloading http://packages/django-stubs/django_stubs-5.0.2-py3-none-any.whl (465 kB)
#6 19.10 Requirement already satisfied: django in /usr/local/lib/python3.10/site-packages (from django-stubs==5.0.2) (4.2.13)
#6 19.10 Requirement already satisfied: asgiref in /usr/local/lib/python3.10/site-packages (from django-stubs==5.0.2) (3.8.1)
#6 19.14 Collecting django-stubs-ext>=5.0.2 (from django-stubs==5.0.2)
#6 19.16   Downloading http://packages/django-stubs-ext/django_stubs_ext-5.1.1-py3-none-any.whl (9.0 kB)
#6 19.17 Requirement already satisfied: typing-extensions>=4.11.0 in /usr/local/lib/python3.10/site-packages (from django-stubs==5.0.2) (4.12.2)
#6 19.21 Collecting types-PyYAML (from django-stubs==5.0.2)
#6 19.23   Downloading http://packages/types-pyyaml/types_PyYAML-6.0.12.20241230-py3-none-any.whl (20 kB)
#6 19.24 Requirement already satisfied: tomli in /usr/local/lib/python3.10/site-packages (from django-stubs==5.0.2) (2.2.1)
#6 19.24 Requirement already satisfied: sqlparse>=0.3.1 in /usr/local/lib/python3.10/site-packages (from django->django-stubs==5.0.2) (0.5.3)
#6 19.33 Installing collected packages: types-PyYAML, django-stubs-ext, django-stubs
#6 19.58 Successfully installed django-stubs-5.0.2 django-stubs-ext-5.1.1 types-PyYAML-6.0.12.20241230
#6 19.58 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 20.30 Looking in indexes: http://simple/
#6 20.34 Collecting django-stubs-ext==5.0.2
#6 20.45   Downloading http://packages/django-stubs-ext/django_stubs_ext-5.0.2-py3-none-any.whl (8.9 kB)
#6 20.46 Requirement already satisfied: django in /usr/local/lib/python3.10/site-packages (from django-stubs-ext==5.0.2) (4.2.13)
#6 20.46 Requirement already satisfied: typing-extensions in /usr/local/lib/python3.10/site-packages (from django-stubs-ext==5.0.2) (4.12.2)
#6 20.46 Requirement already satisfied: asgiref<4,>=3.6.0 in /usr/local/lib/python3.10/site-packages (from django->django-stubs-ext==5.0.2) (3.8.1)
#6 20.46 Requirement already satisfied: sqlparse>=0.3.1 in /usr/local/lib/python3.10/site-packages (from django->django-stubs-ext==5.0.2) (0.5.3)
#6 20.51 Installing collected packages: django-stubs-ext
#6 20.51   Attempting uninstall: django-stubs-ext
#6 20.51     Found existing installation: django-stubs-ext 5.1.1
#6 20.51     Uninstalling django-stubs-ext-5.1.1:
#6 20.51       Successfully uninstalled django-stubs-ext-5.1.1
#6 20.54 Successfully installed django-stubs-ext-5.0.2
#6 20.54 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 21.20 Looking in indexes: http://simple/
#6 21.25 Collecting docutils==0.21.2
#6 21.27   Downloading http://packages/docutils/docutils-0.21.2-py3-none-any.whl (587 kB)
#6 21.31      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 587.4/587.4 kB 17.6 MB/s eta 0:00:00
#6 21.38 Installing collected packages: docutils
#6 21.82 Successfully installed docutils-0.21.2
#6 21.82 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 22.54 Looking in indexes: http://simple/
#6 22.54 Requirement already satisfied: exceptiongroup==1.2.2 in /usr/local/lib/python3.10/site-packages (1.2.2)
#6 22.60 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 23.31 Looking in indexes: http://simple/
#6 23.45 Collecting factory-boy==3.3.0
#6 23.55   Downloading http://packages/factory-boy/factory_boy-3.3.0-py2.py3-none-any.whl (36 kB)
#6 23.86 Collecting Faker>=0.7.0 (from factory-boy==3.3.0)
#6 23.88   Downloading http://packages/faker/Faker-33.3.0-py3-none-any.whl (1.9 MB)
#6 23.92      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.9/1.9 MB 70.6 MB/s eta 0:00:00
#6 23.97 Collecting python-dateutil>=2.4 (from Faker>=0.7.0->factory-boy==3.3.0)
#6 23.99   Downloading http://packages/python-dateutil/python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
#6 24.02 Requirement already satisfied: typing-extensions in /usr/local/lib/python3.10/site-packages (from Faker>=0.7.0->factory-boy==3.3.0) (4.12.2)
#6 24.05 Collecting six>=1.5 (from python-dateutil>=2.4->Faker>=0.7.0->factory-boy==3.3.0)
#6 24.07   Downloading http://packages/six/six-1.17.0-py2.py3-none-any.whl (11 kB)
#6 24.14 Installing collected packages: six, python-dateutil, Faker, factory-boy
#6 26.29 Successfully installed Faker-33.3.0 factory-boy-3.3.0 python-dateutil-2.9.0.post0 six-1.17.0
#6 26.29 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 27.03 Looking in indexes: http://simple/
#6 27.30 Collecting faker==26.0.0
#6 27.40   Downloading http://packages/faker/Faker-26.0.0-py3-none-any.whl (1.8 MB)
#6 27.45      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 63.0 MB/s eta 0:00:00
#6 27.47 Requirement already satisfied: python-dateutil>=2.4 in /usr/local/lib/python3.10/site-packages (from faker==26.0.0) (2.9.0.post0)
#6 27.47 Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/site-packages (from python-dateutil>=2.4->faker==26.0.0) (1.17.0)
#6 27.53 Installing collected packages: faker
#6 27.53   Attempting uninstall: faker
#6 27.53     Found existing installation: Faker 33.3.0
#6 27.67     Uninstalling Faker-33.3.0:
#6 27.72       Successfully uninstalled Faker-33.3.0
#6 29.51 Successfully installed faker-26.0.0
#6 29.51 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 30.26 Looking in indexes: http://simple/
#6 30.32 Collecting filelock==3.16.1
#6 30.32   Using cached http://packages/filelock/filelock-3.16.1-py3-none-any.whl (16 kB)
#6 30.42 Installing collected packages: filelock
#6 30.45 Successfully installed filelock-3.16.1
#6 30.45 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 31.14 Looking in indexes: http://simple/
#6 31.18 Collecting idna==3.7
#6 31.20   Downloading http://packages/idna/idna-3.7-py3-none-any.whl (66 kB)
#6 31.28 Installing collected packages: idna
#6 31.43 Successfully installed idna-3.7
#6 31.43 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 32.13 Looking in indexes: http://simple/
#6 32.25 Collecting importlib-metadata==8.5.0
#6 32.25   Using cached http://packages/importlib-metadata/importlib_metadata-8.5.0-py3-none-any.whl (26 kB)
#6 32.33 Collecting zipp>=3.20 (from importlib-metadata==8.5.0)
#6 32.34   Using cached http://packages/zipp/zipp-3.21.0-py3-none-any.whl (9.6 kB)
#6 32.42 Installing collected packages: zipp, importlib-metadata
#6 32.47 Successfully installed importlib-metadata-8.5.0 zipp-3.21.0
#6 32.47 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 33.19 Looking in indexes: http://simple/
#6 33.19 Requirement already satisfied: iniconfig==2.0.0 in /usr/local/lib/python3.10/site-packages (2.0.0)
#6 33.27 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 34.01 Looking in indexes: http://simple/
#6 34.05 Collecting jaraco-classes==3.4.0
#6 34.05   Using cached http://packages/jaraco-classes/jaraco.classes-3.4.0-py3-none-any.whl (6.8 kB)
#6 34.10 Collecting more-itertools (from jaraco-classes==3.4.0)
#6 34.10   Using cached http://packages/more-itertools/more_itertools-10.5.0-py3-none-any.whl (60 kB)
#6 34.21 Installing collected packages: more-itertools, jaraco-classes
#6 34.27 Successfully installed jaraco-classes-3.4.0 more-itertools-10.5.0
#6 34.27 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 34.97 Looking in indexes: http://simple/
#6 35.01 Collecting jaraco-context==5.3.0
#6 35.17   Downloading http://packages/jaraco-context/jaraco.context-5.3.0-py3-none-any.whl (6.5 kB)
#6 35.19 Requirement already satisfied: backports.tarfile in /usr/local/lib/python3.10/site-packages (from jaraco-context==5.3.0) (1.2.0)
#6 35.31 Installing collected packages: jaraco-context
#6 35.33 Successfully installed jaraco-context-5.3.0
#6 35.33 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 36.16 Looking in indexes: http://simple/
#6 36.21 Collecting jaraco-functools==4.0.1
#6 36.34   Downloading http://packages/jaraco-functools/jaraco.functools-4.0.1-py3-none-any.whl (9.8 kB)
#6 36.35 Requirement already satisfied: more-itertools in /usr/local/lib/python3.10/site-packages (from jaraco-functools==4.0.1) (10.5.0)
#6 36.46 Installing collected packages: jaraco-functools
#6 36.48 Successfully installed jaraco-functools-4.0.1
#6 36.48 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 37.24 Looking in indexes: http://simple/
#6 37.35 Collecting keyring==25.2.1
#6 37.46   Downloading http://packages/keyring/keyring-25.2.1-py3-none-any.whl (38 kB)
#6 37.48 Requirement already satisfied: jaraco.classes in /usr/local/lib/python3.10/site-packages (from keyring==25.2.1) (3.4.0)
#6 37.48 Requirement already satisfied: jaraco.functools in /usr/local/lib/python3.10/site-packages (from keyring==25.2.1) (4.0.1)
#6 37.48 Requirement already satisfied: jaraco.context in /usr/local/lib/python3.10/site-packages (from keyring==25.2.1) (5.3.0)
#6 37.48 Requirement already satisfied: importlib-metadata>=4.11.4 in /usr/local/lib/python3.10/site-packages (from keyring==25.2.1) (8.5.0)
#6 37.51 Collecting SecretStorage>=3.2 (from keyring==25.2.1)
#6 37.51   Using cached http://packages/secretstorage/SecretStorage-3.3.3-py3-none-any.whl (15 kB)
#6 37.53 Collecting jeepney>=0.4.2 (from keyring==25.2.1)
#6 37.54   Using cached http://packages/jeepney/jeepney-0.8.0-py3-none-any.whl (48 kB)
#6 37.58 Requirement already satisfied: zipp>=3.20 in /usr/local/lib/python3.10/site-packages (from importlib-metadata>=4.11.4->keyring==25.2.1) (3.21.0)
#6 37.97 Collecting cryptography>=2.0 (from SecretStorage>=3.2->keyring==25.2.1)
#6 37.98   Using cached http://packages/cryptography/cryptography-44.0.0-cp39-abi3-manylinux_2_28_x86_64.whl (4.2 MB)
#6 38.02 Requirement already satisfied: more-itertools in /usr/local/lib/python3.10/site-packages (from jaraco.classes->keyring==25.2.1) (10.5.0)
#6 38.02 Requirement already satisfied: backports.tarfile in /usr/local/lib/python3.10/site-packages (from jaraco.context->keyring==25.2.1) (1.2.0)
#6 38.03 Requirement already satisfied: cffi>=1.12 in /usr/local/lib/python3.10/site-packages (from cryptography>=2.0->SecretStorage>=3.2->keyring==25.2.1) (1.17.1)
#6 38.04 Requirement already satisfied: pycparser in /usr/local/lib/python3.10/site-packages (from cffi>=1.12->cryptography>=2.0->SecretStorage>=3.2->keyring==25.2.1) (2.22)
#6 38.12 Installing collected packages: jeepney, cryptography, SecretStorage, keyring
#6 38.48 Successfully installed SecretStorage-3.3.3 cryptography-44.0.0 jeepney-0.8.0 keyring-25.2.1
#6 38.48 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 39.30 Looking in indexes: http://simple/
#6 39.36 Collecting markdown-it-py==3.0.0
#6 39.38   Downloading http://packages/markdown-it-py/markdown_it_py-3.0.0-py3-none-any.whl (87 kB)
#6 39.43 Collecting mdurl~=0.1 (from markdown-it-py==3.0.0)
#6 39.45   Downloading http://packages/mdurl/mdurl-0.1.2-py3-none-any.whl (10.0 kB)
#6 39.60 Installing collected packages: mdurl, markdown-it-py
#6 39.71 Successfully installed markdown-it-py-3.0.0 mdurl-0.1.2
#6 39.71 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 40.50 Looking in indexes: http://simple/
#6 40.50 Requirement already satisfied: mdurl==0.1.2 in /usr/local/lib/python3.10/site-packages (0.1.2)
#6 40.66 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 41.49 Looking in indexes: http://simple/
#6 41.54 Collecting more-itertools==10.3.0
#6 41.56   Downloading http://packages/more-itertools/more_itertools-10.3.0-py3-none-any.whl (59 kB)
#6 41.76 Installing collected packages: more-itertools
#6 41.76   Attempting uninstall: more-itertools
#6 41.76     Found existing installation: more-itertools 10.5.0
#6 41.76     Uninstalling more-itertools-10.5.0:
#6 41.77       Successfully uninstalled more-itertools-10.5.0
#6 41.83 Successfully installed more-itertools-10.3.0
#6 41.84 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 42.92 Looking in indexes: http://simple/
#6 43.01 Collecting nh3==0.2.17
#6 43.16   Downloading http://packages/nh3/nh3-0.2.17-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (777 kB)
#6 43.19      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 777.1/777.1 kB 38.7 MB/s eta 0:00:00
#6 43.35 Installing collected packages: nh3
#6 43.39 Successfully installed nh3-0.2.17
#6 43.39 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 44.13 Looking in indexes: http://simple/
#6 44.18 Collecting nodeenv==1.9.1
#6 44.20   Downloading http://packages/nodeenv/nodeenv-1.9.1-py2.py3-none-any.whl (22 kB)
#6 44.35 Installing collected packages: nodeenv
#6 44.38 Successfully installed nodeenv-1.9.1
#6 44.38 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 45.09 Looking in indexes: http://simple/
#6 45.15 Collecting packaging==24.1
#6 45.17   Downloading http://packages/packaging/packaging-24.1-py3-none-any.whl (53 kB)
#6 45.32 Installing collected packages: packaging
#6 45.32   Attempting uninstall: packaging
#6 45.32     Found existing installation: packaging 24.2
#6 45.33     Uninstalling packaging-24.2:
#6 45.33       Successfully uninstalled packaging-24.2
#6 45.38 Successfully installed packaging-24.1
#6 45.38 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 46.13 Looking in indexes: http://simple/
#6 46.17 Collecting pkginfo==1.10.0
#6 46.19   Downloading http://packages/pkginfo/pkginfo-1.10.0-py3-none-any.whl (30 kB)
#6 46.37 Installing collected packages: pkginfo
#6 46.43 Successfully installed pkginfo-1.10.0
#6 46.43 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 47.17 Looking in indexes: http://simple/
#6 47.17 Requirement already satisfied: pluggy==1.5.0 in /usr/local/lib/python3.10/site-packages (1.5.0)
#6 47.32 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 48.09 Looking in indexes: http://simple/
#6 48.09 Requirement already satisfied: pycparser==2.22 in /usr/local/lib/python3.10/site-packages (2.22)
#6 48.23 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 48.95 Looking in indexes: http://simple/
#6 49.01 Collecting pygments==2.18.0
#6 49.03   Downloading http://packages/pygments/pygments-2.18.0-py3-none-any.whl (1.2 MB)
#6 49.07      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 54.4 MB/s eta 0:00:00
#6 49.23 Installing collected packages: pygments
#6 50.11 Successfully installed pygments-2.18.0
#6 50.11 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 50.90 Looking in indexes: http://simple/
#6 50.93 Collecting pyproject-hooks==1.1.0
#6 50.96   Downloading http://packages/pyproject-hooks/pyproject_hooks-1.1.0-py3-none-any.whl (9.2 kB)
#6 51.12 Installing collected packages: pyproject-hooks
#6 51.12   Attempting uninstall: pyproject-hooks
#6 51.12     Found existing installation: pyproject_hooks 1.2.0
#6 51.12     Uninstalling pyproject_hooks-1.2.0:
#6 51.12       Successfully uninstalled pyproject_hooks-1.2.0
#6 51.15 Successfully installed pyproject-hooks-1.1.0
#6 51.15 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 51.86 Looking in indexes: http://simple/
#6 51.99 Collecting pyright==1.1.369
#6 52.08   Downloading http://packages/pyright/pyright-1.1.369-py3-none-any.whl (18 kB)
#6 52.10 Requirement already satisfied: nodeenv>=1.6.0 in /usr/local/lib/python3.10/site-packages (from pyright==1.1.369) (1.9.1)
#6 52.23 Installing collected packages: pyright
#6 52.27 Successfully installed pyright-1.1.369
#6 52.27 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 53.03 Looking in indexes: http://simple/
#6 53.16 Collecting pytest==8.2.2
#6 53.18   Downloading http://packages/pytest/pytest-8.2.2-py3-none-any.whl (339 kB)
#6 53.22 Requirement already satisfied: iniconfig in /usr/local/lib/python3.10/site-packages (from pytest==8.2.2) (2.0.0)
#6 53.22 Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from pytest==8.2.2) (24.1)
#6 53.22 Requirement already satisfied: pluggy<2.0,>=1.5 in /usr/local/lib/python3.10/site-packages (from pytest==8.2.2) (1.5.0)
#6 53.22 Requirement already satisfied: exceptiongroup>=1.0.0rc8 in /usr/local/lib/python3.10/site-packages (from pytest==8.2.2) (1.2.2)
#6 53.22 Requirement already satisfied: tomli>=1 in /usr/local/lib/python3.10/site-packages (from pytest==8.2.2) (2.2.1)
#6 53.38 Installing collected packages: pytest
#6 53.38   Attempting uninstall: pytest
#6 53.38     Found existing installation: pytest 8.3.4
#6 53.39     Uninstalling pytest-8.3.4:
#6 53.54       Successfully uninstalled pytest-8.3.4
#6 53.84 Successfully installed pytest-8.2.2
#6 53.84 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 54.56 Looking in indexes: http://simple/
#6 54.63 Collecting pytest-codspeed==3.0.0
#6 54.79   Downloading http://packages/pytest-codspeed/pytest_codspeed-3.0.0-py3-none-any.whl (15 kB)
#6 54.80 Requirement already satisfied: cffi>=1.17.1 in /usr/local/lib/python3.10/site-packages (from pytest-codspeed==3.0.0) (1.17.1)
#6 54.80 Requirement already satisfied: filelock>=3.12.2 in /usr/local/lib/python3.10/site-packages (from pytest-codspeed==3.0.0) (3.16.1)
#6 54.80 Requirement already satisfied: pytest>=3.8 in /usr/local/lib/python3.10/site-packages (from pytest-codspeed==3.0.0) (8.2.2)
#6 54.96 Collecting rich>=13.8.1 (from pytest-codspeed==3.0.0)
#6 54.98   Downloading http://packages/rich/rich-13.9.4-py3-none-any.whl (242 kB)
#6 55.01 Requirement already satisfied: pycparser in /usr/local/lib/python3.10/site-packages (from cffi>=1.17.1->pytest-codspeed==3.0.0) (2.22)
#6 55.02 Requirement already satisfied: iniconfig in /usr/local/lib/python3.10/site-packages (from pytest>=3.8->pytest-codspeed==3.0.0) (2.0.0)
#6 55.02 Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from pytest>=3.8->pytest-codspeed==3.0.0) (24.1)
#6 55.02 Requirement already satisfied: pluggy<2.0,>=1.5 in /usr/local/lib/python3.10/site-packages (from pytest>=3.8->pytest-codspeed==3.0.0) (1.5.0)
#6 55.02 Requirement already satisfied: exceptiongroup>=1.0.0rc8 in /usr/local/lib/python3.10/site-packages (from pytest>=3.8->pytest-codspeed==3.0.0) (1.2.2)
#6 55.02 Requirement already satisfied: tomli>=1 in /usr/local/lib/python3.10/site-packages (from pytest>=3.8->pytest-codspeed==3.0.0) (2.2.1)
#6 55.02 Requirement already satisfied: markdown-it-py>=2.2.0 in /usr/local/lib/python3.10/site-packages (from rich>=13.8.1->pytest-codspeed==3.0.0) (3.0.0)
#6 55.02 Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /usr/local/lib/python3.10/site-packages (from rich>=13.8.1->pytest-codspeed==3.0.0) (2.18.0)
#6 55.03 Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /usr/local/lib/python3.10/site-packages (from rich>=13.8.1->pytest-codspeed==3.0.0) (4.12.2)
#6 55.03 Requirement already satisfied: mdurl~=0.1 in /usr/local/lib/python3.10/site-packages (from markdown-it-py>=2.2.0->rich>=13.8.1->pytest-codspeed==3.0.0) (0.1.2)
#6 55.14 Installing collected packages: rich, pytest-codspeed
#6 55.39 Successfully installed pytest-codspeed-3.0.0 rich-13.9.4
#6 55.39 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 56.12 Looking in indexes: http://simple/
#6 56.18 Collecting pytest-django==4.8.0
#6 56.27   Downloading http://packages/pytest-django/pytest_django-4.8.0-py3-none-any.whl (23 kB)
#6 56.29 Requirement already satisfied: pytest>=7.0.0 in /usr/local/lib/python3.10/site-packages (from pytest-django==4.8.0) (8.2.2)
#6 56.29 Requirement already satisfied: iniconfig in /usr/local/lib/python3.10/site-packages (from pytest>=7.0.0->pytest-django==4.8.0) (2.0.0)
#6 56.29 Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from pytest>=7.0.0->pytest-django==4.8.0) (24.1)
#6 56.29 Requirement already satisfied: pluggy<2.0,>=1.5 in /usr/local/lib/python3.10/site-packages (from pytest>=7.0.0->pytest-django==4.8.0) (1.5.0)
#6 56.30 Requirement already satisfied: exceptiongroup>=1.0.0rc8 in /usr/local/lib/python3.10/site-packages (from pytest>=7.0.0->pytest-django==4.8.0) (1.2.2)
#6 56.30 Requirement already satisfied: tomli>=1 in /usr/local/lib/python3.10/site-packages (from pytest>=7.0.0->pytest-django==4.8.0) (2.2.1)
#6 56.44 Installing collected packages: pytest-django
#6 56.48 Successfully installed pytest-django-4.8.0
#6 56.48 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 57.18 Looking in indexes: http://simple/
#6 57.24 Collecting pytest-mock==3.14.0
#6 57.26   Downloading http://packages/pytest-mock/pytest_mock-3.14.0-py3-none-any.whl (9.9 kB)
#6 57.28 Requirement already satisfied: pytest>=6.2.5 in /usr/local/lib/python3.10/site-packages (from pytest-mock==3.14.0) (8.2.2)
#6 57.28 Requirement already satisfied: iniconfig in /usr/local/lib/python3.10/site-packages (from pytest>=6.2.5->pytest-mock==3.14.0) (2.0.0)
#6 57.28 Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from pytest>=6.2.5->pytest-mock==3.14.0) (24.1)
#6 57.28 Requirement already satisfied: pluggy<2.0,>=1.5 in /usr/local/lib/python3.10/site-packages (from pytest>=6.2.5->pytest-mock==3.14.0) (1.5.0)
#6 57.29 Requirement already satisfied: exceptiongroup>=1.0.0rc8 in /usr/local/lib/python3.10/site-packages (from pytest>=6.2.5->pytest-mock==3.14.0) (1.2.2)
#6 57.29 Requirement already satisfied: tomli>=1 in /usr/local/lib/python3.10/site-packages (from pytest>=6.2.5->pytest-mock==3.14.0) (2.2.1)
#6 57.49 Installing collected packages: pytest-mock
#6 57.52 Successfully installed pytest-mock-3.14.0
#6 57.52 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 58.23 Looking in indexes: http://simple/
#6 58.27 Collecting pytest-random-order==1.1.1
#6 58.29   Downloading http://packages/pytest-random-order/pytest_random_order-1.1.1-py3-none-any.whl (11 kB)
#6 58.30 Requirement already satisfied: pytest>=3.0.0 in /usr/local/lib/python3.10/site-packages (from pytest-random-order==1.1.1) (8.2.2)
#6 58.30 Requirement already satisfied: iniconfig in /usr/local/lib/python3.10/site-packages (from pytest>=3.0.0->pytest-random-order==1.1.1) (2.0.0)
#6 58.30 Requirement already satisfied: packaging in /usr/local/lib/python3.10/site-packages (from pytest>=3.0.0->pytest-random-order==1.1.1) (24.1)
#6 58.30 Requirement already satisfied: pluggy<2.0,>=1.5 in /usr/local/lib/python3.10/site-packages (from pytest>=3.0.0->pytest-random-order==1.1.1) (1.5.0)
#6 58.30 Requirement already satisfied: exceptiongroup>=1.0.0rc8 in /usr/local/lib/python3.10/site-packages (from pytest>=3.0.0->pytest-random-order==1.1.1) (1.2.2)
#6 58.31 Requirement already satisfied: tomli>=1 in /usr/local/lib/python3.10/site-packages (from pytest>=3.0.0->pytest-random-order==1.1.1) (2.2.1)
#6 58.46 Installing collected packages: pytest-random-order
#6 58.48 Successfully installed pytest-random-order-1.1.1
#6 58.48 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 59.26 Looking in indexes: http://simple/
#6 59.26 Requirement already satisfied: python-dateutil==2.9.0.post0 in /usr/local/lib/python3.10/site-packages (2.9.0.post0)
#6 59.27 Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/site-packages (from python-dateutil==2.9.0.post0) (1.17.0)
#6 59.43 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 60.16 Looking in indexes: http://simple/
#6 60.20 Collecting readme-renderer==43.0
#6 60.22   Downloading http://packages/readme-renderer/readme_renderer-43.0-py3-none-any.whl (13 kB)
#6 60.23 Requirement already satisfied: nh3>=0.2.14 in /usr/local/lib/python3.10/site-packages (from readme-renderer==43.0) (0.2.17)
#6 60.23 Requirement already satisfied: docutils>=0.13.1 in /usr/local/lib/python3.10/site-packages (from readme-renderer==43.0) (0.21.2)
#6 60.24 Requirement already satisfied: Pygments>=2.5.1 in /usr/local/lib/python3.10/site-packages (from readme-renderer==43.0) (2.18.0)
#6 60.38 Installing collected packages: readme-renderer
#6 60.41 Successfully installed readme-renderer-43.0
#6 60.41 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 61.12 Looking in indexes: http://simple/
#6 61.20 Collecting requests==2.32.3
#6 61.21   Using cached http://packages/requests/requests-2.32.3-py3-none-any.whl (64 kB)
#6 61.21 Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/site-packages (from requests==2.32.3) (3.3.2)
#6 61.22 Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/site-packages (from requests==2.32.3) (3.7)
#6 61.56 Collecting urllib3<3,>=1.21.1 (from requests==2.32.3)
#6 61.56   Using cached http://packages/urllib3/urllib3-2.3.0-py3-none-any.whl (128 kB)
#6 61.57 Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/site-packages (from requests==2.32.3) (2024.6.2)
#6 61.70 Installing collected packages: urllib3, requests
#6 61.85 Successfully installed requests-2.32.3 urllib3-2.3.0
#6 61.86 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 62.59 Looking in indexes: http://simple/
#6 62.63 Collecting requests-toolbelt==1.0.0
#6 62.63   Using cached http://packages/requests-toolbelt/requests_toolbelt-1.0.0-py2.py3-none-any.whl (54 kB)
#6 62.63 Requirement already satisfied: requests<3.0.0,>=2.0.1 in /usr/local/lib/python3.10/site-packages (from requests-toolbelt==1.0.0) (2.32.3)
#6 62.64 Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/site-packages (from requests<3.0.0,>=2.0.1->requests-toolbelt==1.0.0) (3.3.2)
#6 62.64 Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/site-packages (from requests<3.0.0,>=2.0.1->requests-toolbelt==1.0.0) (3.7)
#6 62.64 Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/site-packages (from requests<3.0.0,>=2.0.1->requests-toolbelt==1.0.0) (2.3.0)
#6 62.64 Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/site-packages (from requests<3.0.0,>=2.0.1->requests-toolbelt==1.0.0) (2024.6.2)
#6 62.80 Installing collected packages: requests-toolbelt
#6 62.87 Successfully installed requests-toolbelt-1.0.0
#6 62.87 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 63.63 Looking in indexes: http://simple/
#6 63.67 Collecting rfc3986==2.0.0
#6 63.68   Downloading http://packages/rfc3986/rfc3986-2.0.0-py2.py3-none-any.whl (31 kB)
#6 63.86 Installing collected packages: rfc3986
#6 63.90 Successfully installed rfc3986-2.0.0
#6 63.91 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 64.64 Looking in indexes: http://simple/
#6 64.64 Requirement already satisfied: rich==13.9.4 in /usr/local/lib/python3.10/site-packages (13.9.4)
#6 64.64 Requirement already satisfied: markdown-it-py>=2.2.0 in /usr/local/lib/python3.10/site-packages (from rich==13.9.4) (3.0.0)
#6 64.65 Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /usr/local/lib/python3.10/site-packages (from rich==13.9.4) (2.18.0)
#6 64.65 Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /usr/local/lib/python3.10/site-packages (from rich==13.9.4) (4.12.2)
#6 64.66 Requirement already satisfied: mdurl~=0.1 in /usr/local/lib/python3.10/site-packages (from markdown-it-py>=2.2.0->rich==13.9.4) (0.1.2)
#6 64.81 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 65.51 Looking in indexes: http://simple/
#6 66.34 Collecting ruff==0.5.0
#6 66.36   Downloading http://packages/ruff/ruff-0.5.0-py3-none-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (10.1 MB)
#6 66.44      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.1/10.1 MB 161.6 MB/s eta 0:00:00
#6 66.60 Installing collected packages: ruff
#6 66.81 Successfully installed ruff-0.5.0
#6 66.81 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 67.70 Looking in indexes: http://simple/
#6 67.74 Collecting six==1.16.0
#6 67.75   Downloading http://packages/six/six-1.16.0-py2.py3-none-any.whl (11 kB)
#6 67.92 Installing collected packages: six
#6 67.92   Attempting uninstall: six
#6 67.92     Found existing installation: six 1.17.0
#6 67.92     Uninstalling six-1.17.0:
#6 68.06       Successfully uninstalled six-1.17.0
#6 68.09 Successfully installed six-1.16.0
#6 68.09 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 68.81 Looking in indexes: http://simple/
#6 68.84 Collecting sqlparse==0.5.0
#6 68.86   Downloading http://packages/sqlparse/sqlparse-0.5.0-py3-none-any.whl (43 kB)
#6 69.04 Installing collected packages: sqlparse
#6 69.04   Attempting uninstall: sqlparse
#6 69.05     Found existing installation: sqlparse 0.5.3
#6 69.05     Uninstalling sqlparse-0.5.3:
#6 69.05       Successfully uninstalled sqlparse-0.5.3
#6 69.12 Successfully installed sqlparse-0.5.0
#6 69.12 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 69.87 Looking in indexes: http://simple/
#6 69.93 Collecting tomli==2.1.0
#6 69.96   Downloading http://packages/tomli/tomli-2.1.0-py3-none-any.whl (13 kB)
#6 70.13 Installing collected packages: tomli
#6 70.13   Attempting uninstall: tomli
#6 70.13     Found existing installation: tomli 2.2.1
#6 70.13     Uninstalling tomli-2.2.1:
#6 70.14       Successfully uninstalled tomli-2.2.1
#6 70.17 Successfully installed tomli-2.1.0
#6 70.17 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 70.89 Looking in indexes: http://simple/
#6 70.95 Collecting twine==5.1.1
#6 71.09   Downloading http://packages/twine/twine-5.1.1-py3-none-any.whl (38 kB)
#6 71.10 Requirement already satisfied: pkginfo>=1.8.1 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (1.10.0)
#6 71.10 Requirement already satisfied: readme-renderer>=35.0 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (43.0)
#6 71.10 Requirement already satisfied: requests>=2.20 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (2.32.3)
#6 71.10 Requirement already satisfied: requests-toolbelt!=0.9.0,>=0.8.0 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (1.0.0)
#6 71.10 Requirement already satisfied: urllib3>=1.26.0 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (2.3.0)
#6 71.10 Requirement already satisfied: importlib-metadata>=3.6 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (8.5.0)
#6 71.10 Requirement already satisfied: keyring>=15.1 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (25.2.1)
#6 71.10 Requirement already satisfied: rfc3986>=1.4.0 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (2.0.0)
#6 71.11 Requirement already satisfied: rich>=12.0.0 in /usr/local/lib/python3.10/site-packages (from twine==5.1.1) (13.9.4)
#6 71.11 Requirement already satisfied: zipp>=3.20 in /usr/local/lib/python3.10/site-packages (from importlib-metadata>=3.6->twine==5.1.1) (3.21.0)
#6 71.12 Requirement already satisfied: jaraco.classes in /usr/local/lib/python3.10/site-packages (from keyring>=15.1->twine==5.1.1) (3.4.0)
#6 71.12 Requirement already satisfied: jaraco.functools in /usr/local/lib/python3.10/site-packages (from keyring>=15.1->twine==5.1.1) (4.0.1)
#6 71.12 Requirement already satisfied: jaraco.context in /usr/local/lib/python3.10/site-packages (from keyring>=15.1->twine==5.1.1) (5.3.0)
#6 71.13 Requirement already satisfied: SecretStorage>=3.2 in /usr/local/lib/python3.10/site-packages (from keyring>=15.1->twine==5.1.1) (3.3.3)
#6 71.13 Requirement already satisfied: jeepney>=0.4.2 in /usr/local/lib/python3.10/site-packages (from keyring>=15.1->twine==5.1.1) (0.8.0)
#6 71.13 Requirement already satisfied: nh3>=0.2.14 in /usr/local/lib/python3.10/site-packages (from readme-renderer>=35.0->twine==5.1.1) (0.2.17)
#6 71.13 Requirement already satisfied: docutils>=0.13.1 in /usr/local/lib/python3.10/site-packages (from readme-renderer>=35.0->twine==5.1.1) (0.21.2)
#6 71.13 Requirement already satisfied: Pygments>=2.5.1 in /usr/local/lib/python3.10/site-packages (from readme-renderer>=35.0->twine==5.1.1) (2.18.0)
#6 71.14 Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/site-packages (from requests>=2.20->twine==5.1.1) (3.3.2)
#6 71.14 Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/site-packages (from requests>=2.20->twine==5.1.1) (3.7)
#6 71.14 Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/site-packages (from requests>=2.20->twine==5.1.1) (2024.6.2)
#6 71.15 Requirement already satisfied: markdown-it-py>=2.2.0 in /usr/local/lib/python3.10/site-packages (from rich>=12.0.0->twine==5.1.1) (3.0.0)
#6 71.15 Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /usr/local/lib/python3.10/site-packages (from rich>=12.0.0->twine==5.1.1) (4.12.2)
#6 71.17 Requirement already satisfied: mdurl~=0.1 in /usr/local/lib/python3.10/site-packages (from markdown-it-py>=2.2.0->rich>=12.0.0->twine==5.1.1) (0.1.2)
#6 71.17 Requirement already satisfied: cryptography>=2.0 in /usr/local/lib/python3.10/site-packages (from SecretStorage>=3.2->keyring>=15.1->twine==5.1.1) (44.0.0)
#6 71.21 Requirement already satisfied: more-itertools in /usr/local/lib/python3.10/site-packages (from jaraco.classes->keyring>=15.1->twine==5.1.1) (10.3.0)
#6 71.22 Requirement already satisfied: backports.tarfile in /usr/local/lib/python3.10/site-packages (from jaraco.context->keyring>=15.1->twine==5.1.1) (1.2.0)
#6 71.23 Requirement already satisfied: cffi>=1.12 in /usr/local/lib/python3.10/site-packages (from cryptography>=2.0->SecretStorage>=3.2->keyring>=15.1->twine==5.1.1) (1.17.1)
#6 71.24 Requirement already satisfied: pycparser in /usr/local/lib/python3.10/site-packages (from cffi>=1.12->cryptography>=2.0->SecretStorage>=3.2->keyring>=15.1->twine==5.1.1) (2.22)
#6 71.31 Installing collected packages: twine
#6 71.35 Successfully installed twine-5.1.1
#6 71.35 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 72.08 Looking in indexes: http://simple/
#6 72.12 Collecting types-pyyaml==6.0.12.20240311
#6 72.20   Downloading http://packages/types-pyyaml/types_PyYAML-6.0.12.20240311-py3-none-any.whl (15 kB)
#6 72.36 Installing collected packages: types-pyyaml
#6 72.36   Attempting uninstall: types-pyyaml
#6 72.36     Found existing installation: types-PyYAML 6.0.12.20241230
#6 72.37     Uninstalling types-PyYAML-6.0.12.20241230:
#6 72.37       Successfully uninstalled types-PyYAML-6.0.12.20241230
#6 72.39 Successfully installed types-pyyaml-6.0.12.20240311
#6 72.39 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 73.12 Looking in indexes: http://simple/
#6 73.12 Requirement already satisfied: typing-extensions==4.12.2 in /usr/local/lib/python3.10/site-packages (4.12.2)
#6 73.30 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 74.05 Looking in indexes: http://simple/
#6 74.17 Collecting urllib3==2.2.2
#6 74.20   Downloading http://packages/urllib3/urllib3-2.2.2-py3-none-any.whl (121 kB)
#6 74.44 Installing collected packages: urllib3
#6 74.44   Attempting uninstall: urllib3
#6 74.44     Found existing installation: urllib3 2.3.0
#6 74.45     Uninstalling urllib3-2.3.0:
#6 74.45       Successfully uninstalled urllib3-2.3.0
#6 74.59 Successfully installed urllib3-2.2.2
#6 74.59 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 75.43 Looking in indexes: http://simple/
#6 75.43 Requirement already satisfied: zipp==3.21.0 in /usr/local/lib/python3.10/site-packages (3.21.0)
#6 75.60 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager, possibly rendering your system unusable.It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv. Use the --root-user-action option if you know what you are doing and want to suppress this warning.
#6 DONE 76.0s

#7 [4/5] RUN cd /repo && pytest --collect-only -q
#7 2.270 tests/test_listeners.py::test_can_log_errors
#7 2.270 tests/test_listeners.py::test_can_log_all_traces
#7 2.270 tests/test_listeners.py::test_errors_include_caller
#7 2.270 tests/test_listeners.py::test_can_exclude_with_allowlist
#7 2.270 tests/test_listeners.py::test_can_use_fnmatch_pattern_in_allowlist_model
#7 2.270 tests/test_listeners.py::test_can_use_fnmatch_pattern_in_allowlist_field
#7 2.270 tests/test_listeners.py::test_ignore_context_takes_precedence
#7 2.270 tests/test_listeners.py::test_reverts_to_previous_state_when_leaving_zeal_ignore
#7 2.270 tests/test_listeners.py::test_resets_state_in_nested_context
#7 2.270 tests/test_listeners.py::test_can_ignore_specific_models
#7 2.270 tests/test_listeners.py::test_context_specific_allowlist_merges
#7 2.270 tests/test_listeners.py::test_does_not_run_outside_of_context
#7 2.270 tests/test_listeners.py::test_validates_global_allowlist_model_name
#7 2.270 tests/test_listeners.py::test_validates_global_allowlist_field_name
#7 2.270 tests/test_listeners.py::test_allows_fnmatch_in_global_allowlist
#7 2.270 tests/test_listeners.py::test_validates_local_allowlist_model_name
#7 2.270 tests/test_listeners.py::test_validates_local_allowlist_field_name
#7 2.270 tests/test_listeners.py::test_allows_fnmatch_in_local_allowlist
#7 2.270 tests/test_listeners.py::test_validates_related_name_field_names
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_many_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_many_to_one_iterator
#7 2.270 tests/test_nplusones.py::test_handles_prefetch_instead_of_select_related_in_forward_many_to_one
#7 2.270 tests/test_nplusones.py::test_no_false_positive_when_loading_single_object_forward_many_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_many_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_many_to_one_iterator
#7 2.270 tests/test_nplusones.py::test_no_false_positive_when_calling_reverse_many_to_one_twice
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_one_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_one_to_one_iterator
#7 2.270 tests/test_nplusones.py::test_handles_prefetch_instead_of_select_related_in_forward_one_to_one
#7 2.270 tests/test_nplusones.py::test_no_false_positive_when_loading_single_object_forward_one_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_one_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_one_to_one_iterator
#7 2.270 tests/test_nplusones.py::test_handles_prefetch_instead_of_select_related_in_reverse_one_to_one
#7 2.270 tests/test_nplusones.py::test_no_false_positive_when_loading_single_object_reverse_one_to_one
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_many_to_many
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_many_to_many_iterator
#7 2.270 tests/test_nplusones.py::test_no_false_positive_when_loading_single_object_forward_many_to_many
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_many_to_many
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_many_to_many_iterator
#7 2.270 tests/test_nplusones.py::test_no_false_positive_when_loading_single_object_reverse_many_to_many
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_forward_many_to_many_with_no_related_name
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_in_reverse_many_to_many_with_no_related_name
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_due_to_deferred_fields
#7 2.270 tests/test_nplusones.py::test_detects_nplusone_due_to_deferred_fields_in_iterator
#7 2.270 tests/test_nplusones.py::test_handles_prefetch_instead_of_select_related_with_deferred_fields
#7 2.270 tests/test_nplusones.py::test_has_configurable_threshold
#7 2.270 tests/test_nplusones.py::test_does_nothing_if_not_in_middleware
#7 2.270 tests/test_nplusones.py::test_works_in_web_requests
#7 2.270 tests/test_nplusones.py::test_ignores_calls_on_different_lines
#7 2.270 tests/test_patch.py::test_handles_calling_queryset_many_times
#7 2.270 tests/test_patch.py::test_handles_empty_querysets
#7 2.270 tests/test_patch.py::test_handles_get_with_values
#7 2.270 tests/test_patch.py::test_handles_custom_equality_with_relations
#7 2.270 tests/test_patch.py::test_handles_nested_relation_equality
#7 2.270 tests/test_performance.py::test_performance
#7 2.270 tests/test_signals.py::test_signal_send_message
#7 2.270 
#7 2.270 56 tests collected in 0.19s
#7 DONE 2.8s

#8 exporting to image
#8 exporting layers
#8 exporting layers 2.2s done
#8 writing image sha256:b05a6f7f05fd7d64cfbd777215ca1a2fd5bd6461907d220cfaae1beb5ba034d1 done
#8 naming to docker.io/library/tmp:tmp done
#8 DONE 2.2s
